light_node/src/configuration.rs

```diff
@@ -1122,6 +1129,12 @@ impl Environment {
                         )
                     });
 
+                let context_kv_store_gc = args
+                    .value_of("context-kv-store-gc")
+                    .unwrap_or("true")
```

Comment:
@tizoc @sebastiencs @jurajselep 
I would set false as default, to save the same behavior as we have now,
and if anybody wants GC or later for rolling mode, it has to be changed by startup args,
but default now is **irmin + full/archive mode**

Reply:
@tizoc 
and I would maybe add here some check/panic,
for combination: context-kv-store-gc=true with irmin-only,
just to be supersure

Reply:
@bkontur I wouldn't even document this flag, doesn't make sense to run the in-memory backend with GC disabled because memory will fill up very quickly, it is just a workaround for the tests and not something users should use, ever.